src/ServiceControl.Audit.Persistence.RavenDb5/persistence.manifest

```diff
@@ -9,11 +9,6 @@
       "Name": "ServiceControl.Audit/DBPath",
       "Mandatory": true
     },
-    {
-      "Name": "ServiceControl.Audit/HostName",
-      "Mandatory": true
-
-    },
```

Comment:
Isn't this also used to set up ServiceControl API?

Reply:
Yes but this only controls what the persisted needs to know, the instance it self will still have and use the setting